cmd/kube-controller-manager/app/discovery.go

```diff
@@ -52,3 +53,25 @@ func startEndpointSliceController(ctx ControllerContext) (http.Handler, bool, er
 	).Run(int(ctx.ComponentConfig.EndpointSliceController.ConcurrentServiceEndpointSyncs), ctx.Stop)
 	return nil, true, nil
 }
+
+func startEndpointSliceMirroringController(ctx ControllerContext) (http.Handler, bool, error) {
```

Comment:
For my paranoia's sake can we add this to the TestController_DiscoveryError test in https://github.com/kubernetes/kubernetes/blob/release-1.18/cmd/kube-controller-manager/app/core_test.go#L100?

Reply:
Good call, I've added this and the EndpointSlice controller to that test.